### DISCUSSION TITLE
How can I add an apply button to the filter form?

### DISCUSSION THREAD

**Discussion by majidneshati** [filamentphp/filament#11887] [Help] [answered]
### Package

Table builder

### Package Version

v3.2.46

### How can we help you?

I need the filter form information to be submitted with a button

**Comment by leandrocfe** [comment, marked as answer]
https://filamentphp.com/docs/3.x/tables/filters/getting-started/#deferring-filters

?

**Reply by majidneshati** [reply]
I meant `deferFilters()` exactly.


Thanks for your help 🙏👍
